.github/workflows/rpm.yml

```diff
@@ -63,17 +78,30 @@ jobs:
       - name: Login
         if: "steps.vars.outputs.NEED_LOGIN == 'true'"
         env:
-          GITLAB_REGISTRY_RO_TOKEN: '${{ secrets.GITLAB_REGISTRY_RO_TOKEN }}'
+          GITLAB_RO_TOKEN: '${{ secrets.GITLAB_RO_TOKEN }}'
         run: |
-          docker login registry.icinga.com -u build-docker/sles --password-stdin <<<"$GITLAB_REGISTRY_RO_TOKEN"
+          docker login registry.icinga.com -u github --password-stdin <<<"$GITLAB_RO_TOKEN"
```

Comment:
https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html says that personal access tokens are supposed to be used with the username (even though the username is ignored at the moment, but given that there's a hint about this, I think that this might be changed in the future).